frontend/ui/src/pages/Channels/Providers/Airy/ChatPlugin/sections/CustomiseSection.tsx

```diff
@@ -19,11 +19,13 @@ enum BubbleState {
 
 interface CustomiseSectionProps {
   channelId: string;
-  host: string;
+  host?: string;
 }
 
 export const CustomiseSection = ({channelId, host}: CustomiseSectionProps) => {
   const useLocalState = getUseLocalState(channelId);
+  const defaultHost = 'http://airy.core';
```

Comment:
The default host should probably be the one the web app is running on currently, since "airy.core" is only for development